apps/air-discount-scheme/api/src/environments/environment.ts

```diff
@@ -37,8 +39,8 @@ const prodConfig = {
     admins: process.env.ADMINS,
   },
   identityServerAuth: {
-    issuer: process.env.IDENTITY_SERVER_DOMAIN
-      ? `https://${process.env.IDENTITY_SERVER_DOMAIN}`
+    issuer: isProd
+      ? getStaticEnv('SI_PUBLIC_IDENTITY_SERVER_ISSUER_DOMAIN')
       : '',
     audience: '@vegagerdin.is',
```

Comment:
Why is the prod configuration with this condition? Seems redundant to me.

Reply:
Without it there will be errors when running it locally.

Reply:
`issuer: getStaticEnv('SI_PUBLIC_IDENTITY_SERVER_ISSUER_DOMAIN'),`
gives
`[App] Error	1/23/2022, 5:20:18 PM uncaughtException: document is not defined
ReferenceError: document is not defined
    at getStaticEnv (/home/jgg/Code/island.is/dist/apps/air-discount-scheme/api/webpack:/libs/shared/utils/src/lib/getStaticEnv.ts:5:5)`